rust-toolchain

```diff
@@ -1 +1 @@
-stable
+stable-2020-10-08
```

Comment:
Why do we need this? Is this from nearcore?

Reply:
The reason is that the fix that stabilizes wasmer requires that stable be above a certain version, I think 1.44, so this is just to make sure it uses the most recent version.